tools/cloud-build/provision/legacy-tests.tf

```diff
@@ -0,0 +1,51 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+locals {
+  legacy_tests = {
+    ml-a3-highgpu-slurm = "refs/tags/v1.37.1",
```

Comment:
nit. I think instead of `map` it should be a `list<pair>`  so one can run the same blueprint against multiple versions.